docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,284 @@
+```
+
+## Install Istio
+ 
+
+You can then install Istio by your preferred method (helm, operator etc.). Deploy Istio into your cluster. In this case, I ran the following command to install Istio into my cluster:
+
+```
+istioctl install --set profile=minimal
+```
+
+We need to make sure that Istio injects sidecar proxies into our namespace for our testing. To do so, we need to tell Istio what namespaces to inject sidecars into. We can do that with the following command:
+```
+kubectl label ns <namespace_specified> istio-injection=enabled
+```
+Since I installed NGINX ingress and my application into the same namespace, I specified nginx-ingress with the istio-injection=enabled label on that namespace.
+
+```
+kubectl label namespace nginx-ingress istio-injection=enabled
+```
+ 
+
+Using kubectl we can see that the namespace for our demo (nginx-ingress) now has istio-injection=enabled specified:
+
+```
+kubectl get namespacess -A --show-labels
+
+
+default                Active   28h   <none>
+istio-system           Active   24h   istio-injection=disabled
+kube-node-lease        Active   28h   <none>
+kube-public            Active   28h   <none>
+kube-system            Active   28h   <none>
+kubernetes-dashboard   Active   16h   <none>
+local-path-storage     Active   28h   <none>
+nginx-ingress          Active   27h   istio-injection=enabled
+```
+
+After we have setup and configured Istio, we can then deploy NGINX Plus Ingress as well as our applications that will be part of the service mesh. Istio will now inject sidecar proxies based upon how we have configured Istio (namespace configuration).     
+Now, our deployment will look like the following (with Envoy sidecar proxies).
+
+
+{{< img src="./img/nginx_istio_small.png" alt="NGINX Ingress pod with envoy sidecar." >}}
+
+![nginx-ingress](../../../docs/static/img/nic_istio_small.png)
+
+
+We can now see that after configuring Istio with the necessary pieces needed to install a sidecar proxy into the same pod as NGINX ingress, after we deploy NGINX ingress, there are two containers in the same pod for Nginx Ingress:  one is NGINX-ingress, the other is the istio sidecar proxy.
+
+```
+kubectl get pods -A
+
+NAMESPACE       NAME                                      READY   STATUS    RESTARTS   AGE
+kube-system     coredns-854c77959c-h2vrq                  1/1     Running   0          60m
+kube-system     metrics-server-86cbb8457f-fct86           1/1     Running   0          60m
+kube-system     local-path-provisioner-5ff76fc89d-5hjbl   1/1     Running   0          60m
+istio-system    istiod-7c9c9d46d4-qpgff                   1/1     Running   0          60m
+nginx-ingress   nginx-ingress-5898f94c49-v4jrf            2/2     Running   1          41s
+```
+ 
+
+For our application, we are using two new features released in v1.11. These are avaiable in `VirtualServer/VirtualServeroute`.
+
+Here is our VirtualServer configuration to use with Istio: (note `use-cluster-ip` and `requestHeaders`)

```

Comment:
Now the cluster-ip and requestHeaders comments above seem to make sense.  Maybe the explanation of why they are necessary should be right after this for context.